test/xml/src/test/resources/GetXMLWrappedLists.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<AppleBarrel>
+    <GoodApples>
+        <Apple>Fuji</Apple>
+        <Apple>Gala</Apple>
+    </GoodApples>
+    <BadApples>
+        <Apple>Red Delicious</Apple>
+    </BadApples>
```

Comment:
I absolutely agree with this grouping of apples. Who likes red delicious and how did they get that name?